CMakeLists.txt

```diff
@@ -507,7 +512,9 @@ if(RUN_TESTS)
     test/animationinfo_test.cpp)
 endif()
 
-add_executable(${BIN_TARGET} WIN32 MACOSX_BUNDLE ${devilutionx_SRCS})
+add_library(${LIB_TARGET} STATIC ${devilutionxlib_SRCS})

```

Comment:
No need for LIB_TARGET and TEST_TARGET
We only have BIN_TARGET because for some platforms we have to produce an executable with a different name

Reply:
Done 🙂